README.md

```diff
@@ -97,6 +97,7 @@ First, [fork](https://guides.github.com/activities/forking/) this repository, in
 #### Edit an existing package
 
 * `cd types/my-package-to-edit`
+* If there is a `package.json`, run `npm install`.
```

Comment:
Actually, if the type package references other type packages (e.g. `react-autocomplete` depends on `react`), the user would also have to `npm install` inside each o the dependencies).

How should we document this requirement? Also, how can we make it easier? I imagine it would be tricky to do if a type package had lots of dependencies.

Reply:
Does `npm run test` not run `npm install` on dependencies? I thought it did. If so, the advice should be to use `npm run test`.

Reply:
I didn't know that. I just tried it on `master`:

``` sh-session
$ git clean -dfx && npm install && npm run test
Removing node_modules/
Removing yarn.lock
npm WARN tslint@5.14.0 requires a peer of typescript@>=2.1.0 || >=2.1.0-dev || >=2.2.0-dev || >=2.3.0-dev || >=2.4.0-dev || >=2.5.0-dev || >=2.6.0-dev || >=2.7.0-dev || >=2.8.0-dev || >=2.9.0-dev || >=3.0.0-dev || >= 3.1.0-dev || >= 3.2.0-dev but none is installed. You must install peer dependencies yourself.
npm WARN tsutils@2.29.0 requires a peer of typescript@>=2.1.0 || >=2.1.0-dev || >=2.2.0-dev || >=2.3.0-dev || >=2.4.0-dev || >=2.5.0-dev || >=2.6.0-dev || >=2.7.0-dev || >=2.8.0-dev || >=2.9.0-dev || >= 3.0.0-dev || >= 3.1.0-dev but none is installed. You must install peer dependencies yourself.

added 750 packages from 978 contributors and audited 13368 packages in 27.416s
found 0 vulnerabilities


> definitely-typed@0.0.3 test /Users/oliverjash/Development/DefinitelyTyped
> node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed

Clean data
Clean logs
Clean output
Using local Definitely Typed at /Users/oliverjash/Development/DefinitelyTyped.
Parsing definitions...
Found 5805 packages.
Parsing in parallel...
Error: Unused file /Users/oliverjash/Development/DefinitelyTyped/types/ol/assertionerror/index.d.ts
    at checkAllUsedRecur (/Users/oliverjash/Development/DefinitelyTyped/node_modules/types-publisher/src/lib/definition-parser.ts:422:23)
    at <anonymous>
Error: Parsing failed.
    at fail (/Users/oliverjash/Development/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:311:20)
    at ChildProcess.child.on (/Users/oliverjash/Development/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:299:21)
    at emitNone (events.js:106:13)
    at ChildProcess.emit (events.js:208:7)
    at finish (internal/child_process.js:747:14)
    at _combinedTickCallback (internal/process/next_tick.js:132:7)
    at process._tickCallback (internal/process/next_tick.js:181:9)
```

Reply:
Hmf. I think that's the bug Microsoft/types-publisher#575. After I fix that I think we should just recommend `npm run test`.

Reply:
I'll wait for that to be fixed and then pick this PR up again. Thanks